doc/integrator/ngeo.rst.tmpl

```diff
@@ -6,6 +6,9 @@ ngeo
 Organization
 ------------
 
+Simple and advanced modes
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
 The main page where we can redefine the header is in the file:
```

Comment:
"The main page where we can redefine the header" ? Why are we speaking about the header ?
```suggestion
The main page where we can redefine an interface is in the file:
```